tests/robot-cases/Group9-Content-trust/notaryremove.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/python
+
```

Comment:
We need to investigate how to bypass the user and pwd instead of using the pexcept, so just pending this PR.

CI is now the keeper, we has to make sure that any of new test stable.